### PR TITLE
Add commands for file movement, device time, strings, and app lifecycle

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -44,5 +45,22 @@ commands.getWindowSize = async function (windowHandle = 'current') {
   return await this.executeAtom('get_window_size', []);
 };
 
+commands.getDeviceTime = iosCommands.general.getDeviceTime;
```
Comment (yaphatak) [inline lib/commands/general.js]: does this mean I could have done this https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/gesture.js#L42-L50 with just
`commands.isDrag = iosCommands.gestures.isDrag;` ?

**Comment by yaphatak** [comment]
Do check-in this sooner :). Looks like lot of change in driver.js

File: lib/commands/general.js (hunk 1 above)
Comment (imurchie) [inline lib/commands/general.js]: If that is a method that exists in appium-ios-driver, and is in a file in the `/commands/` tree, then yes. They are exported here: https://github.com/appium/appium-ios-driver/blob/master/lib/commands/index.js#L30-L47

**Comment by imurchie** [comment]
The changes to `driver.js` are just pulling stuff out into `start` and `stop`, which the ios driver methods require.

**Comment by yaphatak** [comment]
Yes. I see that. Sooner that goes in I can merge some of changes I have been working on in same file :)

**Pull request merged**